scripts/imagebuild.sh

```diff
@@ -21,43 +21,56 @@ set -o pipefail
 base_dir="$(cd "$(dirname "$0")/.." ; pwd)"
 dockerfile_dir="${base_dir}/images/federation-v2"
 
+SUBCMD=
+if [[ $# == 1 ]]; then
+    SUBCMD="$1"
+fi
+
+REGISTRY=${REGISTRY:-quay.io}
+REPO=${REPO:-kubernetes-multicluster}
+IMAGE=${REGISTRY}/${REPO}/federation-v2
```

Comment:
If this is what going to be triggered from make and/or available choice for user. this should be overrideable by the user.

Reply:
yes they are overridable, please check above comment.

Reply:
As above.